api/v1alpha1/envoypatchpolicy_types.go

```diff
@@ -17,6 +17,7 @@ const (
 )
 
 // +kubebuilder:object:root=true
+// +kubebuilder:subresource:status
```

Comment:
Not something I'd want to hold up this PR for, but along with this addition, I'd love to see something along the lines of:
```
// +kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.conditions[?(@.type=="Programmed")].reason 
```
so you can see if your patch policy applied successfully at a glance. That's just an example, but you could tweak the selector to your liking to showcase the other possible states of the resource.
On that topic, resource age is also nice to have for the custom resources we're introducing. 
```
// +kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
```

Reply:
thanks, added